util/packaging/homebrew/chapel-main.rb

```diff
@@ -22,6 +22,7 @@ class Chapel < Formula
   depends_on "llvm@14"
   depends_on "python@3.10"
   depends_on "cmake"
+  depends_on "hwloc"
 
```

Comment:
Need `depends_on` jemalloc too?

Reply:
It seems like we should add it (it does depend on it after all).

For whatever reason things build fine without it but don't build fine if it's missing `hwloc`.

Reply:
I would guess that's just since jemalloc happens to be installed on our test machine already.

Reply:
Nah, it would error out even with `hwloc` installed, here: 

https://github.com/chapel-lang/chapel/blob/113ce16f668195babb6b2dc718f4a64a0ae3b16c/util/chplenv/chpl_hwloc.py#L49-L53

That `pkg-config` command would work from the root shell but not whatever environment homebrew puts itself in during the install.